clang/include/clang/CIR/Interfaces/ASTAttrInterfaces.h

```diff
@@ -0,0 +1,50 @@
+//===- ASTAttrInterfaces.h - CIR AST Interfaces -----------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MLIR_INTERFACES_CIR_AST_ATR_INTERFACES_H_
+#define MLIR_INTERFACES_CIR_AST_ATR_INTERFACES_H_
```

Comment:
Nice, this is a cool addition! Can you please clang-format this and all other files? This has space and style issues.

Reply:
fixed in a6c1d8c990aa849c0a88db914e1af2c858259ae0